db/migrations/create_comment_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE comment(id SERIAL PRIMARY KEY, comment VARCHAR(240), tweet_id INTEGER REFERENCES tweets (id), user_id INTEGER REFERENCES users (id));
```

Comment:
I had to create this table after the second table, because it references another table.